crates/feldera-types/src/format/avro.rs

```diff
@@ -106,20 +119,21 @@ pub struct AvroParserConfig {
     #[serde(default)]
     pub update_format: AvroUpdateFormat,
 
-    /// Avro schema used to encode input records.
+    /// Avro schema used to encode all records in this stream, specified as a JSON-encoded string.
     ///
-    /// Specified as a string containing schema definition in JSON format.
-    /// This schema must match precisely the SQL view definition, including
-    /// nullability of columns.
+    /// When this property is set, the connector uses the provided schema instead of
+    /// retrieving the schema from the schema registry. This setting is mutually exclusive
+    /// with `registry_urls`.
     pub schema: Option<String>,
```

Comment:
don't we usually use Object for json stuff?

Reply:
Not sure what's an object.  We could try parsing this into `serde_json::Value`, but this is an escaped string, not a regular piece of JSON, so not sure how to do this exactly, plus the library function for deserializing schemas works with strings.